panoptes_aggregation/batch_aggregation.py

```diff
@@ -0,0 +1,88 @@
+from celery import Celery
+import json
+import pandas as pd
+import os
+import urllib3
+from os import getenv
+
+from panoptes_client import Panoptes, Project, Workflow
+from panoptes_aggregation.workflow_config import workflow_extractor_config, workflow_reducer_config
+from panoptes_aggregation.scripts import batch_utils
+from panoptes_client.panoptes import PanoptesAPIException
+
+import logging
+panoptes_client_logger = logging.getLogger('panoptes_client').setLevel(logging.ERROR)
+
+celery = Celery(__name__)
+celery.conf.broker_url = os.environ.get("CELERY_BROKER_URL", "redis://localhost:6379")
+celery.conf.result_backend = os.environ.get("CELERY_RESULT_BACKEND", "redis://localhost:6379")
+
```

Comment:
Very minor style thing, there should be two blank lines above function and class definitions that are outside a `class` constructor.

Reply:
un-flaked the whole file, thanks 👍